Add code to check for determinism (#7747)
MicrosoftDocs/sql-docs: docs/relational-databases/user-defined-functions/deterministic-and-nondeterministic-functions.md

This article identifies the determinism of built-in system functions and the effect on the deterministic property of user-defined functions when it contains a call to extended stored procedures.

### Determining if a function is deterministic

> You can check whether or not a function is deterministic by checking the "is_deterministic" object property for the function. The example below determines if the function Sales.CalculateSalesTax is deterministic.

Reviewer: ```suggestion
You can check whether a function is deterministic by querying the `is_deterministic` object property for the function. The example below determines if the function `Sales.CalculateSalesTax` is deterministic.

```
